packages/web/src/components/cell/cellTypes.ts

```diff
@@ -198,9 +198,10 @@ export type SuspendingSuccessProps = React.PropsWithChildren<
 }
 
 export type NonSuspenseCellQueryResult<
+  TData = any,
```

Comment:
Ahh, I see this PR already is breaking because of this type change here.

So what we could do is add TData as the last generic type parameter in this PR. Just to get it released in a patch or minor release. 

And then we can follow up with another PR that puts the parameters in the correct order that's breaking and will be released as part of the next major release

Reply:
@Tobbe good idea!
This is fixed now

Reply:
@Tobbe could you confirm this is fixed per your feedback and approve to merge? Thanks!

Reply:
@dthyresson I'll take a look. Thanks for the ping